io/eolearn/io/geometry_io.py

```diff
@@ -13,16 +13,76 @@
 import boto3
 import fiona
 import geopandas as gpd
-from eolearn.core import EOPatch, EOTask, FeatureType
+from eolearn.core import EOPatch, EOTask, FeatureType, FeatureTypeSet
```

Comment:
Pylint complains that `FeatureType` is now an unused import.

Reply:
True, fixed. 

PyCharm doesn't mark it as unused, as it is used to describe feature parameter type in the docs.